spec/tracker_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'tracker.rb'
+
+describe Tracker do
+  subject(:tracker) { described_class.new }
+  let(:transaction) { double :transaction, date: '02/08/2017', amount: 1300 }
+  let(:transaction2) { double :transaction, date: '07/08/2017', amount: -600 }
+
+  it 'is initialized with an empty array of transactions' do
+    expect(tracker.transactions).to be_empty
+  end
+
+  describe '#record_transaction' do
+    it 'adds the transaction to the transactions history' do
+      tracker.record_transaction(transaction)
+      expect(tracker.transactions).to include transaction
```

Comment:
This feels like it's testing state, rather than behaviour.  How does the behaviour of a tracker that has had a transaction added to it change? Could you test for that instead?